For

  https://bugzilla.redhat.com/show_bug.cgi?id=601575

Due to an incorrect version of the patch being posted, some
bogus error & debugging messages were left in the RHEL6 libvirt
tree. This isn't upstream since its a RHEL custom addon.

* src/qemu/qemu_driver.c: Remove bogus VIR_ERROR and
  fprintf() calls
---
 src/qemu/qemu_driver.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/src/qemu/qemu_driver.c b/src/qemu/qemu_driver.c
index 37c6aad..c6e819e 100644
--- a/src/qemu/qemu_driver.c
+++ b/src/qemu/qemu_driver.c
@@ -10499,8 +10499,6 @@ qemuDomainMigrateEncodeCookie(struct qemud_driver *driver,
         return -1;
     *cookielen = strlen(*cookie) + 1;
 
-    VIR_ERROR("Encoded %s %d", *cookie, *cookielen);
-
     return 0;
 }
 
@@ -11390,9 +11388,7 @@ qemudDomainMigratePerform (virDomainPtr dom,
     }
 
     /* Clean up the source domain. */
-    fprintf(stderr, "******************* MIG \n");
     qemudShutdownVMDaemon(driver, vm, 1);
-    fprintf(stderr, "******************* YEEHAAA\n");
     resume = 0;
 
     event = virDomainEventNewFromObj(vm,
-- 
1.6.6.1

